lib/internal/http2/core.js

```diff
@@ -2122,6 +2102,7 @@ function afterOpen(session, options, headers, streamOptions, err, fd) {
 class ServerHttp2Stream extends Http2Stream {
   constructor(session, handle, id, options, headers) {
     super(session, options);
+    handle.owner = this;
```

Comment:
I think it might be better to attach `[kOwner]` in `net.js` (and later switching over to always using that symbol over there)

Generally, it would imo be a good idea to have a standardized way to access the JS wrapper object from a C++ handle, and `handle[kOwner]` might be exactly that